Source/Core/JulianDate.js

```diff
@@ -957,6 +959,30 @@ define([
     };
 
     /**
+     * @private
+     * @param {JulianDate[]} julianDates Array of Julian dates
+     * @returns {Boolean} <code>true</code> if each Julian date in julianDates is a valid Iso8601 date.
+     */
+    JulianDate.validIso8601 = function(julianDates) {
+        //>>includeStart('debug', pragmas.debug);
+        if (!defined(julianDates)) {
+            throw new DeveloperError('julianDates must be defined');
+        }
+        if (!isArray(julianDates)) {
+            throw new DeveloperError('julianDates must be an array');
+        }
+        //>>includeEnd('debug');
+
+        var MINIMUM_VALUE = JulianDate.fromIso8601('0000-01-01T00:00:00Z');
```

Comment:
Not very clean to include this here, since it's available in `Iso8601.js` -- but I can't include it because it'll create a circular dependency.